tokio/src/net/unix/incoming.rs

```diff
@@ -1,5 +1,3 @@
-#![cfg(feature = "async-traits")]
```

Comment:
We no longer want this feature?

Reply:
I plan on going back and doing a full sweep of feature flags.